test/torchaudio_unittest/datasets/speechcommands_test.py

```diff
@@ -53,43 +53,58 @@ class TestSpeechCommands(TempDirMixin, TorchaudioTestCase):
 
     root_dir = None
     samples = []
+    train_samples = []
+    valid_samples = []
+    test_samples = []
 
     @classmethod
-    def setUp(cls):
+    def setUpClass(cls):
```

Comment:
setUp is ran at every method call, and so setUpClass should be used instead as is done for the other dataset.